src/Hook/EnableDisable.php

```diff
@@ -50,6 +50,11 @@ private function onBeforeAll(BeforeAll $event)
         $this->bag->add(
             $this->hookHandler->getJsonHookDataWithDefaults(static::class, $defaults)
         );
+
+        if (count($this->bag->get("disable")) > 0) {
```

Comment:
@hendrikmaus Requests that are supposed to be disabled would have priority. I thought that it's more likely to want something specific disabled than the other way around.